sdk/keyvault/ci.yml

```diff
@@ -1,5 +1,3 @@
-# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
```

Comment:
Could you add this line back for this PR and make sure run `npm install -g azure-track2-js-sdk-release-tools@latest` before you use the release tools next time?